### DISCUSSION TITLE
Docker image

### DISCUSSION THREAD

**Discussion by syamsudotdev** [hyperjumptech/monika#44] [Ideas]
Monika docker image for easy installation

**Comment by newm4n** [comment]
One approach....
Make monika, when start, if the json config file argument is not provided, monika should check  for environment variable "MONIKA_JSON_CONFIG", this environment variable is expected to contain full path to monika's configuration json.

So, when we start monika's docker, 

```
docker run --name myMonika -e "MONIKA_JSON_CONFIG=/path/to/container/config/monika.json" -v "/path/to/localconfig:/path/to/container/config"  --detach hyperjump/monika:latest
```

**Reply by sapiderman** [reply]
Monika's Docker support already covered [documentation here](https://monika.hyperjump.tech/tutorial/run-in-docker). 
Also check the hub.docker.com and look for hyperjump/monika
